drivers/win7/amd64/cpu.asm

```diff
@@ -407,7 +407,13 @@ _swsmi PROC FRAME
     xchg rdi, [r10+30h]  ; //rdi value
 
     ; this OUT instruction will write WORD value (smi_code_data) to ports 0xB2 and 0xB3 (SW SMI control and data ports)
-    out 0B2h, ax ; 0xB2
+    push rax
+    .pushreg rax
+    .endprolog

```

Comment:
` .endprolog` seems to be wrong as this is hardly the end of the prologue. Also there's one before in line 394 already.

Reply:
Do you want me to make the changes or get this merged so you can make the updates?

Reply:
Either way works for me. But I'd say, at least the `.endprolog` should be dropped before this gets merged.